### PR TITLE
JA: add site/ja/tutorials/load_data/pandas_dataframe.ipynb

### PR DESCRIPTION
**Pull request by yyone**
original document:
[https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/pandas_dataframe.ipynb](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/pandas_dataframe.ipynb)

### CODE REVIEW

**Comment by yyone** [comment]
@googlebot I signed it!

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 1]
```diff
@@ -0,0 +1,488 @@
...
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "このチュートリアルでは、pandas のDataFrameをロードして、tf.data.Dataset にデータを読み込む例を示します。\n",
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: tf.data.Dataset should be emphasized to consistent with the original document. (e.g. \`tf.data.Dataset\`)

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 2]
```diff
@@ -0,0 +1,488 @@
...
+   "outputs": [],
+   "source": [
+    "df.dtypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "dataframe 内で唯一の object 型である 「thal」列を離散値に変換します。"
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: `object` and `thal` should be emphasized in the same way as original text.

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "WWRhH6r4xxQu"
+   },
+   "source": [
+    "## tf.data.Dataset を使ってデータをロードする"
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: `tf.data.Dataset` should be emphasized to consistent with original document.

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 4]
```diff
@@ -0,0 +1,488 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "tf.data.Dataset.from_tensor_slices メソッドを使って、pandas の dataframeから値を読み込みます。\n",
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: `tf.data.Dataset.from_tensor_slices` should be emphasized to consistent with original document.

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 5]
```diff
@@ -0,0 +1,488 @@
...
+    "\n",
+    "tf.data.Dataset を使う利点は、シンプルに使えて、かつ、大変効率的なデータパイプラインを構築できることです。詳しくは [loading data guide](https://www.tensorflow.org/guide/data) を参照してください。"
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: `tf.data.Dataset` should be emphasized.

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 6]
```diff
@@ -0,0 +1,488 @@
...
+   "source": [
+    "for feat, targ in dataset.take(5):\n",
+    "  print ('Features: {}, Target: {}'.format(feat, targ))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "pd.Series は \\_\\_array\\_\\_ プロトコルを実装しているため、np.array や tf.Tensor を使うところでは、だいたいどこでも使うことができます。"
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: `pd.Series`, `__array__`, `np.array` and `tf.Tensor` should be emphasized.

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 7]
```diff
@@ -0,0 +1,488 @@
...
+    "model_func.compile(optimizer='adam',\n",
+    "                   loss='binary_crossentropy',\n",
+    "                   metrics=['accuracy'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "tf.data を使うときに、pandas の DataFrame の列構造を保持する一番簡単な方法は、DataFrame を辞書型データに変換して、先頭を切り取ることです。"
```
Comment (nuka137) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: [Question]

I could not find the original documents for `先頭を切り取る`. Is it a loose translation in this context?

Comment (yyone) [reply site/ja/tutorials/load_data/pandas_dataframe.ipynb]: Yes.
I thought "先頭を切り取る" was better than "スライス" when translating the original "slice that dictionary"

**Comment by yyone** [comment]
@nuka137 thank you for your review.
I will fix it ASAP.

**Comment by yyone** [comment]
@googlebot I signed it!

**Comment by yyone** [comment]
@googlebot I fixed it.

**Comment by yyone** [comment]
I fixed it without transration of "先頭を切り取る".

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb (hunk 7 above)
Comment (nuka137) [reply site/ja/tutorials/load_data/pandas_dataframe.ipynb]: Thanks, I got it.

**Comment by nuka137** [comment]
LGTM!

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 8]
```diff
@@ -0,0 +1,488 @@
...
+    "colab_type": "text",
+    "id": "Oqa952X4wQKK"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/load_data/pandas_dataframe\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```
Comment (lamberta) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/ja/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
```

Make sure to point the button links to your translation in GitHub :)

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 9]
```diff
@@ -0,0 +1,488 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```
Comment (lamberta) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]:
```suggestion
    "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/ja/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
```

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 10]
```diff
@@ -0,0 +1,488 @@
...
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```
Comment (lamberta) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]:
```suggestion
    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/ja/tutorials/load_data/pandas_dataframe.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```

File: site/ja/tutorials/load_data/pandas_dataframe.ipynb [hunk 11]
```diff
@@ -0,0 +1,488 @@
...
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```
Comment (lamberta) [inline site/ja/tutorials/load_data/pandas_dataframe.ipynb]: Please add a text cell that contains the community transition disclaimer. See the beginning of this notebook as an example: https://github.com/tensorflow/docs/blob/master/site/ja/tutorials/quickstart/beginner.ipynb

**Pull request merged**